src/DynamoCoreWpf/Controls/ShortcutToolbar.xaml.cs

```diff
@@ -53,7 +53,7 @@ public ShortcutToolbar(DynamoViewModel dynamoViewModel)
             var shortcutToolbar = new ShortcutToolbarViewModel(dynamoViewModel);
             DataContext = shortcutToolbar;
             authManager = dynamoViewModel.Model.AuthenticationManager;
-            if (authManager.IsLoggedIn())
+            if (authManager.IsLoggedInSingle())
```

Comment:
If user logged in from splash screen, this would report the wrong state?

Reply:
> If user logged in from splash screen, this would report the wrong state?

keeping it updated here https://github.com/DynamoDS/Dynamo/pull/14070/commits/cb1b0e83bee750f0313fdf4025c07825e64531af